src/SamplesApp/SamplesApp.UITests/TestFramework/ImageAssert.cs

```diff
@@ -315,23 +266,23 @@ string WithContext(string message = null, Action<StringBuilder> builder = null)
 		#endregion
 
 		#region HasPixels
-		public static void HasPixels(FileInfo screenshot, params ExpectedPixels[] expectations)
+		public static void HasPixels(FileInfo actual, params ExpectedPixels[] expectations)
 		{
 			var isSuccess = true;
 			var result = new StringBuilder();
 
-			using (var bitmap = new Bitmap(screenshot.FullName))
+			using (var bitmap = new Bitmap(actual.FullName))
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Refactor this method to add validation of parameter 'actual' before using it.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5363624)